packages/next/src/client/components/router-reducer/fill-lazy-items-till-leaf-with-head.ts

```diff
@@ -14,8 +17,8 @@ export function fillLazyItemsTillLeafWithHead(
   existingCache: CacheNode | undefined,
   routerState: FlightRouterState,
   cacheNodeSeedData: CacheNodeSeedData | null,
-  head: React.ReactNode,
-  prefetchEntry?: PrefetchCacheEntry
```

Comment:
this change is to avoid accidentally passing the wrong argument since they got the similar types, to ensure you always pass the args correctly